elastic/security/workflows/hosts/8.json

```diff
@@ -446,7 +446,7 @@
                 "include_unmapped": true
               },
               {
-                "field": "message",
+                "field": "kubernetes.event.message",
```

Comment:
The query in the page uses the field "message".  We should not change it here.  If you need it changed for LogsDB testing perhaps you should create a new track.

Reply:
DUplicating a track is alot more work...I would do that only if there is no other way to proceed.

Reply:
Also this is not for a one time test...the plan is to have it running in nightlies so that we can see how introducing logsdb improvements reflects on the track (indexing throughput, query latency and storage footprint).